eth/db/chain.py

```diff
@@ -395,6 +395,22 @@ def get_transaction_index(self, transaction_hash: Hash32) -> Tuple[BlockNumber,
         transaction_key = rlp.decode(encoded_key, sedes=TransactionKey)
         return (transaction_key.block_number, transaction_key.index)
 
+    def get_transaction_receipt(self,
```

Comment:
I think this function name and signature should mirror the existing `ChainDB.get_transaction_by_index` function, meaning that rather than a `header` it should operate on a `block_number` and `receipt_index`.